### DISCUSSION TITLE
data is not persisted after pod restarted.

### DISCUSSION THREAD

**Discussion by Goutami-Gowda** [apache/kvrocks#1262] [Q&A]
hello, i created kvrocks statefulset with /var/lib/kvrocks volume. but after pod restart , all the namespace that i have added or gone. how can i persist this?

**Comment by git-hulk** [comment]
Hi @Goutami-Gowda Thanks for your feedback. Those namespaces were stored in kvrocks.conf, so we need to mount this config file as well. The workaround can copy the kvrocks.conf to /var/lib/kvrocks, then load config from /var/lib/kvrocks/kvrocks.conf in [Dockerfile](https://github.com/apache/incubator-kvrocks/blob/unstable/Dockerfile#L40).

**Reply by git-hulk** [reply]
@Goutami-Gowda Can see this PR #1264
